.github/workflows/windows-non-wsl-workflow.yml

```diff
@@ -0,0 +1,51 @@
+name:  Flank WSL
```

Comment:
I think this is the not-WSL job? :) Maybe all this `Flank Windows` instead?

Reply:
will change this. Good spot.